res/css/views/settings/tabs/user/_HelpUserSettingsTab.pcss

```diff
@@ -28,4 +28,9 @@ limitations under the License.
             margin-bottom: $spacing-16;
         }
     }
-}
+
+    div .mx_CopyableText {
```

Comment:
I recommend adding a comment saying what this is for.

Reply:
Hello @richvdh, should I add a comment in the file itself?

Reply:
it seems like you have, thanks.